MaxText/configs/base.yml

```diff
@@ -61,6 +61,13 @@ save_config_to_gcs: False
 dtype: "bfloat16"
 quantization: "" #defaults to no quantization, i.e. bf16. possible alternative setting is 'int8' or use fp8 to run with 8-bit floating-point GeMMs on NVIDIA GPUs.
 quantize_kvcache: False
+# Valid kv_quant_axis values:
+#   - "" is valid only when quantize_kvcache is False
+#   - "dkv" over kv_dimensions
```

Comment:
Can we be be more descriptive on what `kv_dimensions` is?

Wondering if a separate writeup/README (with simple images or other points)outside of this code would be useful to explain the differences. Such as  on `dkv` vs `heads_and_dkv` perf/accuracy tradeoff.